src/kvstore/RocksEngine.cpp

```diff
@@ -459,9 +665,14 @@ nebula::cpp2::ErrorCode RocksEngine::flush() {
   rocksdb::FlushOptions options;
   rocksdb::Status status = db_->Flush(options);
   if (status.ok()) {
+    status = refdb_->Flush(options);
```

Comment:
I think refdb flush is not dependent on db flush.
even if db flush is a fail, refdb can flush.